apps/cowswap-frontend/src/modules/tradeQuote/services/fetchAndProcessQuote.ts

```diff
@@ -43,7 +47,13 @@ export async function fetchAndProcessQuote(
       return
     }
 
-    tradeQuoteManager.onResponse(data, fetchParams)
+    tradeQuoteManager.onResponse(
+      {
+        quoteResults: (data as any).swap || (data as any).quoteResults, // TODO fix types
```

Comment:
Need to fix types and save `BridgeQuoteResults` into jotai atom as well, see `BridgeQuoteAndPost['bridge']` in SDK